### ISSUE TITLE
Agent Node Panic: Try to view a container in the STOP state on the agent node.

### ISSUE THREAD

**Issue by john5du** [amir20/dozzle#3166]
### 🔍 Check for existing issues

- [X] Completed

### How is Dozzle deployed?

Agents

### 📦 Dozzle version

v8.1.2

### ✅ Command used to run Dozzle

```yaml
services:
  dozzle-agent:
    container_name: dozzle-agent
    image: amir20/dozzle:latest
    command: agent
    environment:
      - DOZZLE_HOSTNAME=name
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    ports:
      - 7007:7007
  ```

### 🐛 Describe the bug / provide steps to reproduce it
Try to view a container in the STOP state on the agent node.

### 💻 Environment

```
ubuntu@localhost:~$ sudo docker system info
Client: Docker Engine - Community
 Version:    27.0.3
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.15.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.28.1
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 13
  Running: 9
  Paused: 0
  Stopped: 4
 Images: 12
 Server Version: 27.0.3
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: ae71819c4f5e67bb4d5ae76a6b735f29cc25774e
 runc version: v1.1.13-0-g58aa920
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
 Kernel Version: 5.4.0-72-generic
 Operating System: Ubuntu 20.04.6 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 1.941GiB
 Name: localhost
 ID: xxx
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
```

### 📸 If applicable, add screenshots to help explain your bug
![image](https://github.com/user-attachments/assets/620fa7b8-d68e-48a4-87dd-cdee8038e820)

_No response_

### 📜 If applicable, attach your Dozzle logs. You many need to enable debug mode. See https://dozzle.dev/guide/debugging.
```
time="2024-07-30T11:04:08Z" level=info msg="Dozzle agent version v8.1.2"
time="2024-07-30T11:04:08Z" level=info msg="Agent listening on 0.0.0.0:7007"
time="2024-07-30T17:06:50Z" level=info msg="stopped collecting container stats"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xacd8ed]
goroutine 2210 [running]:
github.com/amir20/dozzle/internal/agent.(*server).FindContainer(0x163cf30?, {0xde7c80?, 0xc0004b1990?}, 0x49cf65?)
	/dozzle/internal/agent/server.go:192 +0x16d
github.com/amir20/dozzle/internal/agent/pb._AgentService_FindContainer_Handler({0xde7c80, 0xc0003f8f00}, {0x10922a8, 0xc000707350}, 0xc000293200, 0x0)
	/dozzle/internal/agent/pb/rpc_grpc.pb.go:384 +0x1a6
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000ff200, {0x10922a8, 0xc0007072c0}, {0x1097b80, 0xc000764480}, 0xc00073e240, 0xc0003f8f90, 0x164e2b8, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.65.0/server.go:1379 +0xdf8
google.golang.org/grpc.(*Server).handleStream(0xc0000ff200, {0x1097b80, 0xc000764480}, 0xc00073e240)
	/go/pkg/mod/google.golang.org/grpc@v1.65.0/server.go:1790 +0xe8b
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	/go/pkg/mod/google.golang.org/grpc@v1.65.0/server.go:1029 +0x8b
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 2147
	/go/pkg/mod/google.golang.org/grpc@v1.65.0/server.go:1040 +0x125

```
_No response_

**Comment by amir20** [comment]
I see the problem.

**Comment by amir20** [comment]
Should be fixed now in #3167. I had forgotten about this even though I knew it was happening. Going to merge and release tomorrow

**Comment by amir20** [comment]
Should be fixed now.
